BrainCloudClient/solution/mono/UnitTests/UnitTests.csproj

```diff
@@ -199,6 +199,9 @@
     <Compile Include="..\..\..\tests\TestTournament.cs">
       <Link>TestTournament.cs</Link>
     </Compile>
+    <Compile Include="..\..\..\tests\TestCustomEntityService.cs">
+      <Link>TestCustomEntityService.cs</Link>
+    </Compile>
     <Compile Include="TestItemCatalog.cs" />
```

Comment:
this is a mistake Ryan left behind we'll want to remove this line and confirm everything still runs and passes 
<Compile Include="TestItemCatalog.cs" />

Reply:
<Compile Include="TestItemCatalog.cs" />      remove this line